gen/field.go

```diff
@@ -473,17 +473,23 @@ func (f fieldGroupGenerator) PrimitiveAccessors(g Generator) error {
 			<$fname := goName .>
 			<reserveFieldOrMethod $fname>
 			<reserveFieldOrMethod (printf "Get%v" $fname)>
-			<if and (not .Required) (isPrimitiveType .Type)>
 			// Get<$fname> returns the value of <$fname> if it is set or its
 			// zero value if it is unset.
```

Comment:
I know this is from before but the comment is not quite right since we return the default value too when it's unset.

Can we make the comment mention that the default will be returned if unset. Something like,

    // Get<$fname> returns the value of <$fname> if it is set or its
    // <if .Default>default<else>zero<end> value if it is unset.

Reply:
Sounds good, I'll update it here.